lib/keyboard-navigation.js

```diff
@@ -37,3 +37,17 @@ export const useFocusTrap = () => {
   }, []);
   return { first, last };
 };
+
+export const onArrowKeys = (e, index, options) => {
+  if (!options.length) return null;
+
+  let offset = 0;
+  if (e.key === 'ArrowUp' || e.key === 'ArrowLeft') {
```

Comment:
What's the rationale for supporting both left and right here?

Reply:
When I was prototyping these components, I also had a [segmented button group](http://power-passenger.glitch.me/#StorySegmentedButton) and a [popover actions list](http://power-passenger.glitch.me/#StoryPopoverMenu_Actions_List) that used keyboard navigation. I was unsure if it made sense for some of these to operate left-right while others operated up-down (since a user won't know how a form element is oriented if they can't see it) so i made all of them work with both.